build_tools/ci/cpu_comparison/run.py

```diff
@@ -38,6 +38,30 @@ def run_conv_test(config, aie_compilation_flags, filename, n_repeats):
     return True
 
 
+class RunningParams(ABC):
```

Comment:
```suggestion
class RunParams(ABC):
```
```suggestion
class LoweringParams(ABC):
```
```suggestion
class TestParams(ABC):
```
Not a native English speaker, but "RunningParams" sounds a bit weird to me, so some suggestions for alternatives.

Reply:
Makes sense. I chose "TestParams" and have updated it.